<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseWordWindow</class>
 <widget class="QDialog" name="ChooseWordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="firstWordButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Word1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="secondWordButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Word2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="thirdWordButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Word3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
